src/sc.c

```diff
@@ -1520,6 +1520,9 @@ sc_snprintf (char *str, size_t size, const char *fmt, ...)
     return;
   }
 
+  /* avoid uninitialized bytes if strlen (src) < size - 1 */
+  memset (str, (int) '\0', size);
```

Comment:
There is no cast needed, a simple 0 (just the number, no backslash) is best with memset (in general as well).

Reply:
Okay, I removed the cast and set the parameter to `0`.

Reply:
Thanks. I removed such casts in three other places as well.